java/ql/src/experimental/CWE-094/ScriptEngine.qhelp

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p> The ScriptEngine api is available since the release of Java 6.
+It allows application to interact with script written in language such as JavaScript.</p>
```

Comment:
```suggestion
<p> The ScriptEngine API has been available since the release of Java 6.
It allows applications to interact with scripts written in languages such as JavaScript.</p>
```